docs/architecture/blazor-for-web-forms-developers/project-structure.md

```diff
@@ -246,6 +246,20 @@ To run the Blazor WebAssembly app, choose one of the following approaches:
 
 Blazor WebAssembly apps can be debugged in both browser and Visual Studio.See [Debug ASP.NET Core Blazor WebAssembly](/aspnet/core/blazor/debug) for details.
 
+Both Blazor Server and Blazor WebAssembly apps support Hot Reload in Visual Studio. Hot Reload is a feature that automatically updates changes made to a Blazor app live, in the browser. You can toggle whether Hot Reload is enabled from its icon in the toolbar:
+
+![Hot Reload in Visual Studio 2022](media/project-structure/hot-reload-1.png)
+
+Clicking the caret beside the icon reveals additional options. You can toggle Hot Reload on or off, restart the application, and toggle whether Hot Reload should occur whenever a file is saved.
+
+![Hot Reload menu in Visual Studio 2022](media/project-structure/hot-reload-2.png)
+
+You can also access additional configuration options. The configuration dialog lets you specify whether hot reload should be enabled when debugging (along with Edit and Continue), when starting without debugging, or when a file is saved.
+
+![Hot Reload configuration options](media/project-structure/hot-reload-3.png)
+
+With Hot Reload, the "developer inner loop" has been greatly streamlined. Without Hot Reload, a Blazor developer would typically need to restart and rerun the app after every change, navigating to the appropriate part of the app as required. With Hot Reload, changes can be made to the running app without the need to restart in most cases. Hot Reload even retains the state of pages, so there's no need to have to re-enter form values or otherwise get the app back where you need it.
```

Comment:
```suggestion
The "developer inner loop" has been greatly streamlined with Hot Reload. Using Hot Reload, a Blazor developer would typically need to restart and rerun the app after every change, navigating to the appropriate part of the app as required. With Hot Reload, changes can be made to the running app without the need to restart in most cases. Hot Reload even retains the state of pages, so there's no need to have to re-enter form values or otherwise get the app back where you need it.
```

Three sentences started the exact same way "With Hot Reload", let's try mixing it up a bit.